apps/server/src/modules/task/controller/api-test/task.api.spec.ts

```diff
@@ -1456,4 +1457,256 @@ describe('Task Controller (API)', () => {
 				.expect(501);
 		});
 	});
+
+	describe('Multiple users filtered by assignment', () => {
+		let app: INestApplication;
+		let em: EntityManager;
+		let currentUser: ICurrentUser;
+
+		const createStudent = (id: number) => {
+			const studentRole = roleFactory.build({
+				permissions: [
+					Permission.TASK_DASHBOARD_VIEW_V3,
+					Permission.JOIN_MEETING,
+					Permission.TASK_CARD_VIEW,
+					Permission.TEAM_CREATE,
+					Permission.TEAM_EDIT,
+					Permission.TOOL_CREATE_ETHERPAD,
```

Comment:
for the purpose of these tests, why do we need permissions like JOIN_MEETING? We should have only those permissions strictly needed for this endpoint

Reply:
Not sure about that. The test should reflect the production setting as much as possible.
The permissions here are copied from the seed data which represents the different roles.
If we test with a more restricted set of permissions we might miss errors.